client/state/selectors/get-site-options.js

```diff
@@ -18,11 +18,5 @@ export default ( state, siteId ) => {
 	if ( ! site ) {
 		return null;
 	}
-	let options = site.options || {};
-	const defaultPostFormat = options.default_post_format;
-	// The 'standard' post format is saved as an option of '0'
-	if ( ! defaultPostFormat || defaultPostFormat === '0' ) {
-		options = { ...options, default_post_format: 'standard' };
-	}
-	return options;
+	return site.options || {};
```

Comment:
Doesn't this create a new empty object each time `getSiteOptions` is called? Shouldn't we use some memoization?

Reply:
We could use a static empty object and return the same instance every time. But I think it doesn't matter at all in this case.

`site.options` is supposed to be always there and the default only protects against exception in case the data is corrupted in some way.

Also, the output of `getSiteOptions` is almost always just an input to `getSiteOption` and returning a new object every time doesn't matter either.

Reply:
I added a static `EMPTY_OPTIONS` object and return it as default instead of creating a new object on each call.